docs/AboutUs.md

```diff
@@ -57,3 +57,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 * Responsibilities: UI
+
+### Kim-Chan Tze Yong, Joenz
+
+<img src="images/joenz.png" width="2000px">
```

Comment:
The image seems rather big! Is this intended?

Reply:
Oh no! This is indeed a mistake on my part. Will fix this.